src/donejs-carousel-plugin-test.js

```diff
@@ -79,7 +79,9 @@ describe('tucows-donejs-carousel', () => {
 				},
 				slides: {
 					type: 'observable',
-					value: ['slide1', 'slide2']
```

Comment:
@tracer99 has made me aware that sometimes there is a good reason to leave it like this, esp. for performance reasons. For SSR, because the function is non-deterministic, it has to recalculate its value every time for any person which slows down page speed. for this tests file, i won't bother reverting the modifications.